test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/project.json

```diff
@@ -1,22 +1,18 @@
 {
   "dependencies": {
     "xunit": "2.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-*"
   },
   "frameworks": {
-    "netcoreapp1.0": {
```

Comment:
[This](https://github.com/aspnet/Mvc/pull/4429/files#diff-6cd8ed09f7672665ceb74b469731acacL19) is all I did for the test projects, Adding netstandardapp1.5 to the imports is probably the important part. But that's a temporary fix while we wait for everything to move to netcoreapp1.0, right?